### PR TITLE
Express: use generics for Request

### PR DESCRIPTION
**Pull request by rockwyc992**
In #37718, that missed to export generics for Request, so we must use `Resuest` in `@type/express-serve-static-core` not `@type/express`.

This PR like a syntax sugar to use this easily.

``` typescript
// Example Code
import { Request } from 'express';
//import { Request } from 'express-serve-static-core';

app.get('/:id', (req: Request<{id: int}>) => {
    // do something here.
});
```

---

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: (#37502)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/express/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Type definitions for Express 4.17
 // Project: http://expressjs.com
 // Definitions by: Boris Yankov <https://github.com/borisyankov>
+//                 China Medical University Hospital <https://github.com/CMUH>
```
Comment (uniqueiniquity) [inline types/express/index.d.ts]: It looks like this is an organization, not an individual account, so adding it here will have no effect.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

**Comment by uniqueiniquity** [comment]
@OliverJAsh Could you review this change?

File: types/express/index.d.ts (hunk 1 above)
Comment (rockwyc992) [reply types/express/index.d.ts]: Yes, it's fine. This contribution is own by this organization, not only me.

**Comment by rockwyc992** [comment]
> @rockwyc992 I haven't seen anything from you in a while and this PR currently has problems that prevent it from being merged. The PR will be closed tomorrow if there aren't new commits to fix the issues.

WTF? I have closed that **Change Request** , but this buggy bot didn't know that.
Alright, I will open a new PR to continue.

**Pull request closed without merging**